v2/emailpassword/custom-ui/init/dashboard.mdx

```diff
@@ -4,6 +4,12 @@ title: "Step 4: User management dashboard"
 hide_title: true
 ---
 
+
+import {Redirect} from '@docusaurus/router';
+
+<Redirect to="/docs/emailpassword/custom-ui/init/user-management-dashboard/setup" />
+
+
 # Step 4: User management dashboard
```

Comment:
you can remove all the content from this file.. an dadd copy docs above the `import {Redirect} from '@docusaurus/router';` so it gets copied over to other recipes as well.

Reply:
removed content and added the `Redirector` component.